client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts

```diff
@@ -193,6 +237,8 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
                             patientClinicalDetails.isIsolationProblemMoreInfo : '',
                         isolationStartDate: convertDate(patientClinicalDetails.isolationStartTime),
                         isolationEndDate: convertDate(patientClinicalDetails.isolationEndTime),
+                        isolationSource: patientClinicalDetails.isolationSource !== null?
+                            patientClinicalDetails.isolationSource : '',
```

Comment:
if isolation source is null why not putting -1, isn't isolation source a number?

Reply:
That is true. I did it because the `Select` component considers empty string as no value.

Reply:
Ok so because this field is changed by a `Select` component with predetermined fields, I reached the conclusion that its initial value should be null and it is changed to specific isolation sources